examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3174,11 +3177,28 @@ class GLTFParser {
 
 		}
 
-		// workarounds for mesh and geometry
+		// TODO(mrdoob) Clean up
+
+		if ( material.aoMap ) {
+
+			if ( geometry.attributes.uv2 === undefined ) {
+
+				if ( material.aoMap.userData._gltfTexCoord !== 0 && geometry.attributes.uv !== undefined ) {
 
-		if ( material.aoMap && geometry.attributes.uv2 === undefined && geometry.attributes.uv !== undefined ) {
+					geometry.setAttribute( 'uv2', geometry.attributes.uv );
 
-			geometry.setAttribute( 'uv2', geometry.attributes.uv );
+				}
+
+			} else {
+
+				if ( material.aoMap.userData._gltfTexCoord === 0 ) {
+
+					console.warn( 'THREE.GLTFLoader: Removed uv2 attribute as it\'s not being used.' );
+					geometry.deleteAttribute( 'uv2' );
+
+				}
+
+			}
```

Comment:
I think this would do it:

```js
if ( material.aoMap ) {

	if ( material.aoMap.userData._gltfTexCoord === 0 && geometry.attributes.uv !== undefined ) {

		if ( geometry.attributes.uv2 !== undefined ) {

			console.warn( 'THREE.GLTFLoader: Removed unused uv2 attribute.' );

		}

		geometry.setAttribute( 'uv2', geometry.attributes.uv );

	}

}
```